source/Outfit.cpp

```diff
@@ -289,6 +286,21 @@ void Outfit::Load(const DataNode &node)
 			child.PrintTrace("Skipping unrecognized attribute:");
 	}
 
+	// If no plural name has been defined, append an 's' to the name and use that.
+	// If the name ends in an 's', and no plural name has been defined, print a
+	// warning since an explicit plural name is always required in this case.
+	// Unless this outfit definition isn't declared with the `outfit` keyword,
+	// because then this is probably being done in `add attributes` on a ship,
+	// so the name doesn't matter.
+	if(pluralName.empty())
+	{
+		pluralName = name + 's';
+		if(name.back() == 's' && node.Token(0) == "outfit")
+			node.PrintTrace("Warning: outfit \"" + name +
+					"\" requires an explicit plural name definition, but none is provided. Defaulting to \""
+					+ pluralName + "\".");
```

Comment:
So, we default to not adding an `s`, fine.
Then you don't need to explicitly define the plural name as often, so you remove those entries?
Then you get a load of warnings because there is no explicit definition even though the default is correct. So, you add the explicit plural definitions back? To explicitly set it to the default?

Reply:
To be clear, I don't agree with having a warning in the first place. My preferred solution is this without the warning.

If we're going with your way, (which, I'm always willing to compromise) there is no reason to default to something wrong.

Reply:
I think if we're going to not add the `s` for words ending with `s`, there should be a warning.

Reply:
Thats what this suggestion is, yeah

Reply:
In either case of adding or not adding the "s", we're throwing an error, and a content creator's immediate action should be to address the error, so it's largely inconsequential if we add the "s" or not. I can see why we'd go either way on adding or not adding the "s", so I just coin flipped it and the coin chose to not add it.